README.md

```diff
@@ -1,2 +1,11 @@
-# tranquility-example-cucumber
-an example of [tranquility](https://github.com/tranquility-bdd/tranquility-bdd) + cucumber
+[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
+
+# tranquility-cucumber-example
+
+an example of [tranquility](https://github.com/tranquility-bdd/tranquility) + cucumber for golang ([godog](https://github.com/DATA-DOG/godog))
+
+## how to build & run (temporary)
+
+1. clone the repo
+2. `$ go get github.com/DATA-DOG/godog`
```

Comment:
Shouldn't we add also go get github.com/tranquility-bdd/tranquility

Reply:
improved, thanks!